sdk/python/kfp/v2/google/client/schedule.py

```diff
@@ -41,7 +41,8 @@ def create_from_pipeline_file(
     pipeline_path: str,
     schedule: str,
     project_id: str,
-    region: str = 'us-central1',
```

Comment:
This seems to be a breaking change.
Perhaps we should keep the original naming.